docs/make.jl

```diff
@@ -13,7 +13,8 @@ makedocs(
   modules = [Documenter, JRRTMGP],
   pages = Any[
     "Home" => "index.md",
-    "Radiative Transfer Equation (RTE)" => "RTE.md",
+    "RTE" => "RTE.md",
+    "RRTMGP" => Any[ "gas optics rrtmgp" => "RRTMGP/mo_gas_optics_rrtmgp.md" ],
```

Comment:
```suggestion
    "RRTMGP" => Any[ "gas optics" => "RRTMGP/mo_gas_optics_rrtmgp.md" ],
```
I think we can remove `rrtmgp` from the title here, since it's already under `rrtmgp`.